src/ScssAssetConverter.php

```diff
@@ -25,8 +25,22 @@ class ScssAssetConverter extends Component implements AssetConverterInterface
      * significantly degrade the performance.
      */
     public $forceConvert = false;
-
-    private $compiler;
+    
+    public $formatter = null;
+    
+    protected $scssPhpFormatterMap =
+    [
+        'compact'       => '\Leafo\ScssPhp\Formatter\Compact',
+        'compressed'    => '\Leafo\ScssPhp\Formatter\Compressed',
+        'crunched'      => '\Leafo\ScssPhp\Formatter\Crunched',
+        'debug'         => '\Leafo\ScssPhp\Formatter\Debug',
+        'expanded'      => '\Leafo\ScssPhp\Formatter\Expanded',
+        'nested'        => '\Leafo\ScssPhp\Formatter\Nested',
```

Comment:
I would suggest changing `compact`, `compressed`, etc. to class constants. This way the IDE autocomplete can help the user.

So for example the following code would be valid:
```php
new ScssAssetConverter([
    'formatter' => ScssAssetConverter::FORMAT_COMPACT
]);
```

Also, please use `\Leafo\ScssPhp\Formatter\Compact::class` instead of `'\Leafo\ScssPhp\Formatter\Compact'`. (And others.) The reason for this is that it helps static analysis tools to know what's going on.

Reply:
I have already move to constant